server/game/cards/01_SOR/bases/SecurityComplex.ts

```diff
@@ -0,0 +1,23 @@
+import { BaseCard } from "../../../core/card/BaseCard";
+import AbilityHelper from "../../../AbilityHelper";
+
+export default class SecurityComplex extends BaseCard {
+    protected override getImplementationId () {
+        return {
+            id: '2429341052',
+            internalName: 'security-complex',
+        };
+    }
+
+    public override setupCardAbilities () {
+        this.setEpicActionAbility({
+            title: 'Give a Shield token to a non-leader unit',
+            targetResolver: {
+                cardCondition: (card) => card.isNonLeaderUnit(),
```

Comment:
We have some dedicated filters to make it a little simpler to check stuff like card type, zone, etc. In this case it would be `cardTypeFilter: WildcardCardType.NonLeaderUnit`